src/settings/FeeFinesTable/Owners.js

```diff
@@ -41,7 +43,7 @@ const Owners = ({ dataOptions, onChange, filterShared }) => {
     <div>
       <Row>
         <Col xs>
-          <span style={{ fontSize: 'large', fontWeight: '600' }}>
+          <span className={css.singularMesage}>
```

Comment:
Typo, so nothing is being applied here. I wonder whether `<Headline>` or `<Label>` is more appropriate anyway?